### ISSUE TITLE
Inconsistent documentation about pending mandates

### ISSUE THREAD

**Issue by nphmuller** [mollie/api-documentation#455]
The docs describing how to create subscriptions mention that a subscription can be created when a mandate status is either `valid` or `pending`.
(https://docs.mollie.com/payments/recurring#charging-periodically-with-subscriptions)

The mandates API reference however mentions that a mandate status is set to pending when no first payment has been made.
(https://docs.mollie.com/reference/v2/mandates-api/get-mandate#response)

Since the first link mentions that there should be a valid first payment, but also that a mandate status can be pending (which happens when there is no first payment), this seems like an inconsistency.

The pending status can be reproduced the following way:
- Create payment with test key and properties: RecurringType.First and Method.CreditCard.
- Go to url in response
- Choose: cancel
- The mandate status is now pending

If the app assumes the status `pending` is valid, the app will think that it's user with a cancelled first payment to be a valid and create a subscription for it. This gives the app's user unauthorized access to paid features until the next subscription payment date fails (will it will, because the first payment is cancelled).

**Comment by MollieRick** [comment]
Hi @nphmuller,

Thanks for your message and sorry for the radio silence for so long!
I'll dive into it and update you when I know more.

**Comment by nphmuller** [comment]
@MollieRick No problem! I've had a conversation about this with Mollie Support via email. Might be worth checking out. Ticket nr: 291962

**Comment by MollieRick** [comment]
@nphmuller Almost a week later but we just deployed a change to the way we process mandates. All (new) mandates will get the status `invalid` when the first payment fails. This include the statuses `canceled`, `failed`, and `expired`. I changed the docs regarding this change and hope that it will make it easier for integrating mandates and subscriptions in your system.

Thanks again for reaching out. If you find more inconsistencies, don't hesitate to reach out to use via our tech support or via this repository. :-)
